src/script/conversation/ConversationRepository.ts

```diff
@@ -1711,10 +1711,26 @@ export class ConversationRepository {
   }
 
   public async sendTypingStart(conversationEntity: Conversation) {
+    /*
+      Currently typing endpoint is not implemented on backend for federated environments
+      @todo: remove this condition when backend is ready and api-client in packages is updated to support domain in typing endpoint
+    */
+    const isFederated = this.core.backendFeatures?.isFederated;
```

Comment:
Maybe declare this after constructor ?

```suggestion
    this.isFederated = this.core.backendFeatures?.isFederated;
```

Reply:
<img width="879" alt="image" src="https://user-images.githubusercontent.com/63250054/201917680-0c6ca944-22a0-45cd-b7f4-8bf495db6bf0.png">
we're in a method and it has nothing to do with the main class so why this.?

Reply:
I commented after review it :)